package.json

```diff
@@ -21,7 +21,8 @@
     "lint": "eslint --ext .ts,.js . && prettier -c src lib test stubs",
     "lint:fix": "eslint --fix --ext .ts,.js . && prettier -w src lib test stubs",
     "release": "pnpm build && pnpm test && changelogen --release --push && npm publish",
-    "test": "pnpm lint && vitest run --coverage"
+    "test": "pnpm lint && vitest run --coverage",
+    "test:bun": "JITI_DEBUG=1 bun test/bun/index.ts"
```

Comment:
The `bun --bun` flag could be added here to make sure we're using bun througout.

Reply:
Thanks for the tip. just did :)